src/stylesheets/panel.scss

```diff
@@ -47,14 +51,23 @@
     background: darken($red, 10);
   }
 }
+.panel-instruction .panel-heading {
+  background: darken($blue-tint, 10);
+  &:hover {
+    background: $blue;
+  }
+}
 
 .panel-default .panel-heading {
   &:hover {
     background: lighten($gray, 10);
   }
 }
 
+.static-panel {
+  pointer-events: none;
```

Comment:
Okay, found the issue to https://github.com/NYPL-Simplified/circulation-web/pull/221#issuecomment-478717463. When you add this you disable any interaction with the form inside the panel. I'm not sure why you would want no pointer events in a static panel. I'm guessing you're assuming it's static content? In any case, this does not work.

Reply:
It was the fastest way to disable the style changes when you hover over the header (and yes, I was assuming static content).  Will fix!

Reply:
Better to fix things a better way than a fast way.